jasmine/spec/inverted-index-test.js

```diff
@@ -75,10 +75,10 @@ describe('Index', function() {
     });
 
     it('should return -No such word found- if a string not in the books was passed.', function() {
-      expect(index.searchIndex('searchterm')).toBe(-1);
-      expect(index.searchIndex('hellterm')).toBe(-1);
-      expect(index.searchIndex('hallo')).toBe(-1);
-      expect(index.searchIndex('Esther.')).toBe(-1);
+      expect(index.searchIndex('searchterm')).toBe(null);
```

Comment:
**Code Quality - Nitpick**
If a word not in the Index is searched it should return null.